src/Uno.UI/UI/Xaml/Media/TransformGroup.cs

```diff
@@ -104,15 +127,42 @@ internal override Matrix3x2 ToMatrix(Point absoluteOrigin)
 				foreach (var child in Children)
 				{
 					matrix *= child.ToMatrix(absoluteOrigin);
-					absoluteOrigin = new Point(
```

Comment:
d'oh.

Reply:
Yeah when I did transforms I thought it was logic to also transform the origin before applying the next transform ... but actually I was wrong 😶